build.ps1

```diff
@@ -108,7 +108,7 @@ try {
 
         # Run cleanup first so the samples don't get overloaded with old devices
         RunApp provisioning\Samples\service\CleanupEnrollmentsSample "Provisioning\Service\CleanupEnrollmentsSample"
-        RunApp iot-hub\Samples\service\CleanUpDevicesSample "IoTHub\Service\CleanUpDevicesSample"
+        RunApp iot-hub\Samples\service\CleanupDevicesSample "IoTHub\Service\CleanupDevicesSample" "-i ""$env:IOTHUB_CONNECTION_STRING"" -s ""$env:STORAGE_ACCOUNT_CONNECTION_STRING"" -p ""$env:PATH_TO_DEVICE_PREFIX_FOR_DELETION_FILE"""

```

Comment:
Passing environment variable values as parameters?

Reply:
yes, these values are available to the pipeline as environment variables (from keyvault): https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/vsts/vsts.yaml#L47